script/script.js

```diff
@@ -73,24 +75,48 @@ const datatable = {
     ],
     data: smallFilmSet
 };
+const clearFieldsAndMessages = function () {
+    webix.confirm({
+        text: "Do you still want to clear all fields?"
+    }).then(
+        (response) => {
+            if (response) {}
```

Comment:
I think it should be:

> if(response){
    $$("form").clear();
    $$("form").clearValidation();
}

Reply:
is corrected, ready for review

Reply:
actually you don't need to check for response, because if a confirm promise is resolved, we will always `true`  result in this handler: https://snippet.webix.com/ym3vkb2s